### PR TITLE
Feature lwdev 8786 prevent multiple same time selfie processing

### PR DESCRIPTION
**Pull request by olegtrofimenko**

### CODE REVIEW

File: client/Lykke.Service.ResourceLocker.Client/IResourceLockerApi.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using Lykke.Service.ResourceLocker.Client.Models;
+using Refit;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.ResourceLocker.Client
+{
+    public interface IResourceLockerApi
+    {
+        [Post("/api/lock/lockresource")]
+        Task<LockedResourceResponse> LockResourceAsync([Body] LockedResourceRequest request);
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/IResourceLockerApi.cs]: Add description for methods and arguments

File: client/Lykke.Service.ResourceLocker.Client/IResourceLockerClient.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using Lykke.Service.ResourceLocker.Client.Models;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.ResourceLocker.Client
+{
+    public interface IResourceLockerClient
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/IResourceLockerClient.cs:6]: Why there are two public interfaces with the same names?

File: client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.csproj [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Lykke.Service.ResourceLocker.Client.xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Lykke.Common" Version="7.0.1" />
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.csproj]: Update nugets in the solution

File: client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.xml [hunk 4]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.xml]: Delete the file from git

File: client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceRequest.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Lykke.Service.ResourceLocker.Client.Models
+{
+    public class LockedResourceRequest
+    {
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceRequest.cs]: Add description

File: client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceResponse.cs [hunk 6]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Lykke.Service.ResourceLocker.Client.Models
+{
+    public class LockedResourceResponse
+    {
+        public string Key { get; set; }
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceResponse.cs]: Add description

File: client/Lykke.Service.ResourceLocker.Client/Models/ReleaseResourceRequest.cs [hunk 7]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.Service.ResourceLocker.Client.Models
+{
+    public class ReleaseResourceRequest
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.ResourceLocker.Client/Models/ReleaseResourceRequest.cs]: Add description

File: settings.txt [hunk 8]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (MikhailGerasimov) [inline settings.txt]: Delete this file

File: src/Lykke.Service.ResourceLocker/Models/LockedResourceModel.cs [hunk 9]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.Service.ResourceLocker.Models
+{
+    public class LockedResourceModel
```
Comment (MikhailGerasimov) [inline src/Lykke.Service.ResourceLocker/Models/LockedResourceModel.cs]: Add description

**Review by MikhailGerasimov — COMMENTED** [review]
Add settings.yaml https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/477691938/New+Settings+Service

File: client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.csproj (hunk 3 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.csproj]: updated

File: client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.xml (hunk 4 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/Lykke.Service.ResourceLocker.Client.xml]: done

File: client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceRequest.cs (hunk 5 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceRequest.cs]: done

File: client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceResponse.cs (hunk 6 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/Models/LockedResourceResponse.cs]: done

File: client/Lykke.Service.ResourceLocker.Client/Models/ReleaseResourceRequest.cs (hunk 7 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/Models/ReleaseResourceRequest.cs]: done

File: settings.txt (hunk 8 above)
Comment (olegtrofimenko) [reply settings.txt]: done

File: src/Lykke.Service.ResourceLocker/Models/LockedResourceModel.cs (hunk 9 above)
Comment (olegtrofimenko) [reply src/Lykke.Service.ResourceLocker/Models/LockedResourceModel.cs]: done

File: client/Lykke.Service.ResourceLocker.Client/IResourceLockerApi.cs (hunk 1 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/IResourceLockerApi.cs]: done

File: client/Lykke.Service.ResourceLocker.Client/IResourceLockerClient.cs (hunk 2 above)
Comment (olegtrofimenko) [reply client/Lykke.Service.ResourceLocker.Client/IResourceLockerClient.cs:6]: done

**Pull request merged**